src/gui/qgsfontbutton.cpp

```diff
@@ -869,6 +869,7 @@ void QgsFontButton::updatePreview( const QColor &color, QgsTextFormat *format, Q
       context.setUseAdvancedEffects( true );
       context.setFlag( QgsRenderContext::Antialiasing, true );
       context.setPainter( &p );
+      context.setFlag( QgsRenderContext::Antialiasing, true );
```

Comment:
You set the antialiasing flag twice over three lines, should one setFlag() go away?

Reply:
Ah, bad rebase sorry, fixed